src/scr.c

```diff
@@ -1058,6 +1290,10 @@ static int scr_get_params()
         __FILE__, __LINE__
       );
     }
+    if (scr_my_rank_world == 0) {
+      // 47. Dump SCR parameters - SCR_FLUSH_ASYNC_PERCEN
+      scr_dbg(scr_debug, "SCR_FLUSH_ASYNC_PERCEN = %f ", scr_flush_async_percent);
```

Comment:
Another typo in the name here.  Just be sure to double check all of the names are spelled correctly.  For most params, we want to use the name verbatim.  That helps in case someone would like to cut-and-paste this output into a file if needed.
```suggestion
      scr_dbg(scr_debug, "SCR_FLUSH_ASYNC_PERCENT = %f ", scr_flush_async_percent);
```